peer/network_test.go

```diff
@@ -719,28 +716,22 @@ type HelloGossip struct {
 	Msg string `serialize:"true"`
 }
 
-func (h HelloGossip) Handle(handler message.GossipHandler, nodeID ids.NodeID) error {
-	return handler.HandleEthTxs(nodeID, message.EthTxsGossip{})
+func (tx *HelloGossip) GossipID() ids.ID {
+	return ids.FromStringOrPanic(tx.Msg)
 }
 
-func (h HelloGossip) String() string {
-	return fmt.Sprintf("HelloGossip(%s)", h.Msg)
-}
-
-func (h HelloGossip) Bytes() []byte {
-	// no op
-	return nil
+type helloGossipMarshaller struct {
+	codec codec.Manager
 }
 
-type testGossipHandler struct {
-	received bool
-	nodeID   ids.NodeID
+func (g helloGossipMarshaller) MarshalGossip(tx *HelloGossip) ([]byte, error) {
+	return g.codec.Marshal(0, tx)
 }
 
-func (t *testGossipHandler) HandleEthTxs(nodeID ids.NodeID, msg message.EthTxsGossip) error {
-	t.received = true
-	t.nodeID = nodeID
-	return nil
+func (g helloGossipMarshaller) UnmarshalGossip(bytes []byte) (*HelloGossip, error) {
+	h := &HelloGossip{}
+	_, error := g.codec.Unmarshal(bytes, h)
+	return h, error
```

Comment:
nit convention in Go (🤷) is to have error variables as `err`

```suggestion
	_, err := g.codec.Unmarshal(bytes, h)
	return h, err
```